Dockerfile

```diff
@@ -2,5 +2,6 @@ FROM python:2.7-alpine
 RUN pip install requests
 COPY cleanreg.py /cleanreg.py
 COPY LICENSE /LICENSE
+RUN /cleanreg.py -h
```

Comment:
I found it useful to have a simple syntax error check on build-time, by running the script. But yeah, it's bad style to have this empty layer in the app's Dockerimage. I reverted the commit.